packages/remix-gcloud-functions-gen2/server.ts

```diff
@@ -0,0 +1,164 @@
+import type { NextFunction } from "express";
+import type { AppLoadContext, ServerBuild } from "@remix-run/server-runtime";
+import type {
+  Request as GcfRequest,
+  Response as GcfResponse,
+} from "@google-cloud/functions-framework";
+import type { 
+  RequestInit as NodeRequestInit,
+  Response as NodeResponse
+ } from "@remix-run/node";
+import {
+  createRequestHandler as createRemixRequestHandler,
+  Headers as NodeHeaders,
+  Request as NodeRequest,
+  writeReadableStreamToWritable
+} from "@remix-run/node";
+import {
+  // This has been added as a global in node 15+
+  AbortController,
+} from "@remix-run/node";
```

Comment:
See https://github.com/remix-run/remix/pull/2266#discussion_r890304493

```suggestion
import {
  // This has been added as a global in node 15+
  AbortController,
  createRequestHandler as createRemixRequestHandler,
  Headers as NodeHeaders,
  Request as NodeRequest,
  writeReadableStreamToWritable
} from "@remix-run/node";
```